modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts

```diff
@@ -13,14 +13,22 @@ export interface Selection {
 export type ContentFormat = 'raw' | 'text' | 'html' | 'tree';
 
 export interface GetContentArgs {
-  format?: ContentFormat;
-  get?: boolean;
-  content?: string;
+  format: ContentFormat;
+  get: boolean;
+  content: any;
```

Comment:
```suggestion
  content?: any;
```
Remember we found that this wasn't always set in the "before" case, so it should still be optional. Either that or we need to clone the various different types you've added to here as well.